mmv1/templates/terraform/encoders/memorystore_instance.go.erb

```diff
@@ -0,0 +1,27 @@
+v, ok := d.GetOk("desired_psc_auto_connections")
+if !ok {
+    return obj, nil // No desired connections, nothing to update
+}
+l := v.([]interface{})
+req := make([]interface{}, 0, len(l))
+for _, raw := range l {
+    if raw == nil {
+        continue
+    }
+    original := raw.(map[string]interface{})
+    transformed := make(map[string]interface{})
+
+    transformedProjectId := original["project_id"]
```

Comment:
Nit: Could just be `projectId :=` and `network` below. Calling it `transformedProjectId` confused me a bit while reading because I was expecting the value to be transformed in some way.

Reply:
Renamed as per suggestion. 

I had used transformed for psc_auto_connection request object while desired_psc_auto_connection object.  transformedProjectId and transformedNetwork belonged to trnasformed object.